bedrock/externalpages/templates/externalpages/pocket/includes/footer.html

```diff
@@ -0,0 +1,88 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+<footer class="pocket-footer-container">
+    <div class="pocket-footer-content">
+        {# Top section #}
+        <nav class="pocket-footer-primary">
+            <section class="pocket-footer-section">
+                <h6 class="pocket-footer-heading">Products</h6>
+                <ul class="pocket-footer-list">
+                    <li>
+                        <a>Must-read articles</a>
```

Comment:
None of these links have an `href`. Is that something we should fix before merging this?